elasticsearch/src/main/java/org/hibernate/search/elasticsearch/cfg/ElasticsearchEnvironment.java

```diff
@@ -223,6 +224,21 @@
 	 */
 	public static final String SCROLL_TIMEOUT = "elasticsearch.scroll_timeout";
 
+	/**
+	 * Property for specifying the behaviour when Elasticsearch encounters a previously unknown field in a document.
```

Comment:
Could you clarify who is the subject of "the behaviour"?

Also "unknown field" isn't very clear IMO. An "unexpected field name"? Or better, a "field name which was not declared in the index schema" ?